Formula/libcbor.rb

```diff
@@ -0,0 +1,35 @@
+ class Libcbor < Formula
```

Comment:
Wrong indentation, starting from first line.

Reply:
sorry, fixing and rebasing.